app/Http/Requests/TaskCategoryStoreRequest.php

```diff
@@ -25,6 +25,7 @@ public function rules()
     {
         return [
             'name' => 'required|string|max:255',
+            'color' => 'required|min:3|max:7'
```

Comment:
can u make a validation more stronger.
according to color box we use, min length of color-code cannot be 3 it must have 6digits (0x000000 - 0xFFFFFF)